src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -28,21 +41,35 @@ public Arm() {
       */
     }
 	
-    public void setArmPos() {
+    public void setArmGoal(double targetPosition, double targetVelocity) {
       //Sets arm to the optimal angle for amp, speaker and Ground intake | used to score in amp
       //these values are in constants
-      //pass in where scorign and use switch statement
-    
-
+      //pass in where scorign and use switch statement to alternate between each angle needed
+      targetPosition = getArmClampedGoal(targetPosition);
+      
     }
-    
-    public void setSpeed() {
-      //move the arm around based off the right joystick movement on the manipulator joystick
+    public void controllerMoveArm(double rightJoystick) {
+      /*
+      // move the arm around based off the right joystick movement on the manipulator joystick
       //use the trapezoid thingy from robot code 2023
+      //Math below | Summary: Take in controller Y axis as a double then calculate amount of volts needed to pass to the arm and when to stop based off of the controller movement. Done so by finding the constraints of the arm, translating the controller numbers, and finding how many volts and when to stop using feedvolts and PID
```

Comment:
trapezoidal / feedforward / pid control should happen in periodic, not a one-time function. They are _always_ needed for the arm to function, so they should always be running.